pages/common/ffmpeg.md

```diff
@@ -2,6 +2,14 @@
 
 > Video conversion tool.
 
+- Convert a video into a different container format:
```

Comment:
What is meant by "a different container format"? Is this just the wrapper format and not the actual data itself?

Reply:
Exactly. MKV and MP4 are examples of container formats that wrap several different types of audio or video codecs.

Reply:
I guess there is a way to convert the actual data then? Just curious :smiley_cat: